### PR TITLE
Merchant orders

### PR DESCRIPTION
**Pull request by Chulstro**

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -1 +1,2 @@
 --require spec_helper
+--format NyanCatFormatter
```
Comment (kathytbui) [inline .rspec:2]: Yay nyancat!

Comment (Strompy) [inline .rspec:2]: !!!

File: app/controllers/merchant/dashboard_controller.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class Merchant::DashboardController < Merchant::BaseController
 
   def index
-    @merchant = user
+    @merchant = user.merchant
```
Comment (Strompy) [inline app/controllers/merchant/dashboard_controller.rb:4]: Easy fix!

File: app/models/item_order.rb [hunk 3]
```diff
@@ -18,4 +18,11 @@ def default_status
     self.status = 0 if status.nil?
   end
 
+  def set_merchant
+    if !item.nil?
+      item = Item.find(self.item_id)
```
Comment (Strompy) [inline app/models/item_order.rb:23]: Workaround for validations

**Review by Strompy — APPROVED** [review]

**Review by kathytbui — APPROVED** [review]

File: app/models/item_order.rb (hunk 3 above)
Comment (kathytbui) [reply app/models/item_order.rb:23]: This was a great work around

File: app/models/item_order.rb [hunk 4]
```diff
@@ -18,4 +18,11 @@ def default_status
     self.status = 0 if status.nil?
   end
 
+  def set_merchant
+    if !item.nil?
```
Comment (Chulstro) [inline app/models/item_order.rb:22]: Did we just need this line because of the tests?

**Pull request merged**